applications/facings-nft-collection-manager.md

```diff
@@ -0,0 +1,233 @@
+
+
+### Team Org Repos
+
+- https://github.com/facings
+- https://github.com/FACINGS/eospyo
+
+### Team Member Repos
+
+- https://github.com/robrigo
+- https://drive.google.com/drive/folders/1OfEwjO_ho5QTB9dfrbjMk2IUPEZ6iq6m?usp=sharing
+- https://github.com/roadscape
+- https://github.com/felsin
+- https://github.com/mr-souza
+- https://github.com/mewmix
+- https://heyjames.design/
+- https://github.com/mmoreirasantosdev
+
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/robertkonsdorf/
+- https://www.linkedin.com/in/keirkleinknecht/
+- https://www.linkedin.com/in/mr-souza/
+- https://www.linkedin.com/in/marcos-moreira-51285b16/
+
+
+## Development Status
+
+Some initial interface designs are included in the project details above. Beyond that work, our team has spent considerable time researching the landscape and spec’ing components related to building a fully custodial NFT launchpad on top of the AtomicAssets standard, so we understand the problem domain well. The NFT Collection Manager is a self-custody and simpler version of the product we originally set out to build. Our interest in building this is to fill the void we see in robust collection management solutions, to provide a well-scoped utility for the EOS ecosystem to start benefiting from, and to ship an MVP that we can start getting feedback and traction with in the short term.
+
+
+## Development Roadmap
+
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):** 4 FTE
+- **Total Costs:** 80,000 USD
+
+
+### Milestone 1 — Account Login & View Collections
+
+- **Estimated duration:** 1 month
+- **FTE:**  4
+- **Costs:** 40,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | GPLv3 |
+| 0b. | Documentation | We will provide a ‘Getting Started’ guide, deployment instructions, as well as inline documentation of the code. |
+| 0c. | Testing Guide | Documentation for a walkthrough of the UI as described and the intended outcome of each subset will be provided along with expected feedback results. FACINGS UI standards including visibility of system status, user control and freedom, consistency, error prevention, recognition over recall, flexibility and efficiency, minimalistic design, error recovery and documentation will all be implemented. |
+| 0d. | Docker | We will provide a Dockerfile that can be used to launch a UI instance and test all the functionality delivered with this milestone. |
+| 1. | Base Implementation | Home page: show featured collections, “about” section, and login link. |
```

Comment:
How are "featured collections" managed? Is this data sourced from a shared contract on chain? If so please include which EOSIO chains have a compatible deployment of the shared contract.

Reply:
Our intention with the “featured collections” area on the main page is to provide a sense of life and activity in the app, to create an engaging home page. This could be as simple as showing random chosen, whitelisted Atomic collections to render for now. As far as which data source, the Atomic API would be expected.